### DISCUSSION TITLE
dane godzinowe

### DISCUSSION THREAD

**Discussion by robertfili** [PiotrMachowski/Home-Assistant-custom-components-Tauron-AMIplus#88] [Q&A]
Witam,
Mam tylko pytanie, czy dla laika jak ja jest w zasięgu skonfigurowanie danych godzinowych w HA i jak to zrobić krok po kroku.
Jest zgrubny opis w FAQ ale dla mnie delikatnie mówiąc niewystarczający. W szczeglności nie wiem jak ugryść "tauron_importer positions".

"How to display hourly data in Energy dashboard?

To show hourly data in Energy dashboard you have to use tauron_importer positions."

**Comment by PiotrMachowski** [comment]
Po prostu normalnie dodajesz integrację, a potem przy wyborze źródeł danych do panelu Energia zamiast encji musisz wybrać pozycje, których identyfikatory zaczynają się od "tauron_importer"

**Comment by mmarcines** [comment]
@robertfili jak od "0" konfigurujesz panel Eneria to jest przewodnik / wizard, ktory Cie poprosi o podanie encji do konsumpcji i produkcji. Musisz wybrac importer (ale u mnie musze zaczac wpisywac "Tauron AMIplus ..." i woczas z listy wybrac importer consumption / generation. (@PiotrMachowski  masz ew. pojecie dlaczego nie wyszukuje encji jak wpiszemy slowo kluczowe importer?) I zwracam uwage (na co mi rowniez zwrocono) - panel energia nie wyswietli danych za "dzis" bo tauron aktualizuje dane raz dziennie. W zwiazku z tym patrz w panel energia na dzien wczorajszy / ostatni tydzien / ostatni miesiac.

![image](https://user-images.githubusercontent.com/75376143/213868077-a096afb6-4910-427f-94ee-b6d662bf1be2.png)

**Reply by PiotrMachowski** [reply]
Najwyraźniej HA wyszukuje tylko po nazwach, a nie po ID. Na to nie poradzę

**Comment by krzyzak44** [comment]
![Screenshot_20230121_140531_Home Assistant](https://user-images.githubusercontent.com/73403351/213868162-7fd22404-9dc5-4460-9635-ef71c75d54a6.jpg)

**Comment by mmarcines** [comment]
![image](https://user-images.githubusercontent.com/75376143/213868194-f4926f79-52ec-401f-8702-eb7cdb8d9305.png)

**Comment by mmarcines** [comment]
![image](https://user-images.githubusercontent.com/75376143/213868267-01d177f3-54ee-4c4c-8ded-a4589d1e4a06.png)
